Tocca.js

```diff
@@ -47,7 +47,8 @@
       dbltapThreshold: win.DBL_TAP_THRESHOLD || 200, // delay needed to detect a double tap
       longtapThreshold: win.LONG_TAP_THRESHOLD || 1000, // delay needed to detect a long tap
       tapPrecision: win.TAP_PRECISION / 2 || 60 / 2, // touch events boundaries ( 60px by default )
-      justTouchEvents: win.JUST_ON_TOUCH_DEVICES
+      justTouchEvents: win.JUST_ON_TOUCH_DEVICES,
+      namespace: win.NAMESPACE || '',
```

Comment:
could you please explain the use of namespaced events? I am not aware of any DOM implementation for the events supported by `Tocca.js`.

Reply:
Do you want the explanation in the PR?

Reply:
The namespacing would be if there were implementations of `swipeleft` added to future versions of browsers.  You could namespace the event (prefix it with `tocca-` so it's `tocca-swipeleft` and that way it's future proofed

Reply:
It's an optional config that helps to avoid naming collisions.

Reply:
There are at moment no native DOM events that overlap with the `Tocca.js` namespace. In case one day they will be added there will be no need to use `Tocca.js` anymore. In this case this feature is not necessary and doesn't solve any issue